lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -10,103 +10,74 @@
 public class Lesson6 {
 
   public static void main(String[] args) {
-    var application = new SpringApplication(Lesson6.class);
-    application.run(args);
+    SpringApplication.run(Lesson6.class, args);
   }
 
-  /**
-   * Returns true if the provided age meets the minimum US voting age and false otherwise.
-   *
-   * @param age The age to check.
-   * @return True if the age corresponds to a voting age and false otherwise.
-   */
   public static boolean canVote(int age) {
-    return false;
+    return age >= 18;
   }
 
-  /**
-   * Compares two strings lexographically.
-   *
-   * @param value1 The first `String` to compare.
-   * @param value2 The second `String` to compare.
-   * @rerturn -1 if a is less than b, 1 if a is greater than b, and 0 otherwise.
-   */
-  public static int compareStrings(String a, String b) {
-    // The distance will be a number less than 0 if string `a` is lexographically less than `b`, 1
-    // if it is greater, and 0 if the strings are equal.
-    int distance = Helpers.computeLexographicDistance(a, b);
-
-    // TODO(you): Finish this method.
-
-    return 0;
+  public static int compareStrings(String value1, String value2) {
+    return value1.compareTo(value2);
```

Comment:
Defeats the purpose of the exercise, try again.

Reply:
I changed the method used for returning the String for the exercise